### PR TITLE
Several misc keys

### PR DESCRIPTION
**Pull request by hannob**

### CODE REVIEW

**Comment by BenBE** [comment]
1. Please also document the encrypted Wikipedia key (password `test`).
2. Include the CSR for the report-uri key (also that key might be worth putting into the software test directory).

**Comment by hannob** [comment]
The password-protected wikipedia key is identical to the unprotected one, thus it would just be a duplicate if we included that.

File: src/misc/wikipedia-pkcs8.meta [hunk 1]
```diff
@@ -0,0 +1,2 @@
+uri = https://en.wikipedia.org/wiki/PKCS_8
+description = Example key from Wikipedia page about PKCS #8
```
Comment (BenBE) [inline src/misc/wikipedia-pkcs8.meta:2]:
```suggestion
description = Example key from Wikipedia page about PKCS #8
password = test
```

I noticed those keys are identical, thus just documenting that password suffices.

**Review by BenBE — COMMENTED** [review]
Why not move the report-uri files over to `src/softwaretests/report-uri`?

**Comment by hannob** [comment]
My intention with the "softwaretest" dir was more of it being files shipped with software to test.
The reporturi key is a bit different, as it's a test key for an online tool (that no longer exists though). But either way, if you prefer I can move it. Ultimately I mainly care to have the keys somewhere in the repo.

**Comment by BenBE** [comment]
> My intention with the "softwaretest" dir was more of it being files shipped with software to test. The reporturi key is a bit different, as it's a test key for an online tool (that no longer exists though). But either way, if you prefer I can move it. Ultimately I mainly care to have the keys somewhere in the repo.

No, just wanted to be sure I understand the placement correctly. With that explanation it's fine where it is.

**Pull request merged**